product_abc_classification/models/abc_classification_product_level.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models
```
Replace `_()` by `self.env._()` as well.